src/encord_active/lib/labels/classification.py

```diff
@@ -0,0 +1,98 @@
+import uuid
+from dataclasses import asdict, dataclass, field
+from typing import List, Set, Union
+
+from encord.objects.common import NestableOption, RadioAttribute
+from encord.objects.ontology_structure import Classification, OntologyStructure
+from encord.orm.label_row import LabelRow
+
+from encord_active.lib.common.time import get_timestamp
+
+
+@dataclass
+class LabelClassification:
+    name: str
+    value: str
+    createdAt: str
+    createdBy: str
+    confidence: float
+    featureHash: str
+    manualAnnotation: bool
+
+
+@dataclass
+class Answer:
+    name: str
+    value: str
+    featureHash: str
+
+
+@dataclass
+class ClassificationAnswer:
+    name: str
+    value: str
+    featureHash: str
+    manualAnnotation: bool
+    answers: List[Answer] = field(default_factory=list)
+
+
+def build_label_classification(classification: Classification) -> LabelClassification:
+    attr = classification.attributes[0]
+    return LabelClassification(
+        name=attr.name,
+        value=attr.name.lower(),
+        createdAt=get_timestamp(),
+        createdBy="robot@cord.tech",
+        confidence=1.0,
+        featureHash=classification.feature_node_hash,
+        manualAnnotation=True,
+    )
+
+
+def build_classification_answer(
+    label_classification: LabelClassification, option: NestableOption, attribute: RadioAttribute
+):
+    return ClassificationAnswer(
+        name=label_classification.name,
+        value=label_classification.value,
+        featureHash=attribute.feature_node_hash,
+        answers=[Answer(name=option.label, value=option.value, featureHash=option.feature_node_hash)],
+        manualAnnotation=True,
+    )
+
+
+def update_label_row_with_classification(
+    label_row: LabelRow, classification: Classification, image_class: str
+) -> LabelRow:
+    attribute = classification.attributes[0]
+    classification_hash = str(uuid.uuid4())[:8]
+    data_hash = list(label_row["data_units"].keys())[0]
```

Comment:
Seems like this only works for single images? Should we perhaps at least raise an error of more data units are available?

Reply:
This is consistent with the way we import data units in the `init` command (we don't create image groups).
 
But you're right, an error would be helpful here in case we start importing more than one data unit

Reply:
Alternatively, we can add a `data_hash` optional argument and when that is empty we just add the class to all data units.